typekit/Opaques.cpp

```diff
@@ -13,11 +13,19 @@
 
 void orogen_typekits::toIntermediate(::wrappers::PCLVertices& intermediate, ::pcl::Vertices const& real_type)
 {
-    intermediate.vertices = real_type.vertices;
+    size_t size = real_type.vertices.size();
+    intermediate.vertices.resize(size);
+    for(size_t i = 0; i < size; i++) {
+        intermediate.vertices[i] = real_type.vertices[i];
+    }
```

Comment:
This works as well (and probably more likely to result in a memcpy:

    intermediate.vertices.assign(real_type.vertices.begin(), real_type.vertices.end());

Reply:
I'll check if that still compiles, but to not get into repeated inserts(), one should still resize the vector beforehand.